scripts/flitcli/flit_init.py

```diff
@@ -161,7 +159,13 @@ def copy_files(dest_to_src, remove_license=True):
                 litmus_to_copy[os.path.join('tests', srcfile)] = srcpath
         copy_files(litmus_to_copy)
 
+        # No need for Empty test if other litmus tests are being used
+        to_copy.pop('tests/Empty.cpp')
+
     return flit_update.main(['--directory', args.directory])
 
+    # Finish copying remaining files, after determining if Empty test is needed
+    copy_files(to_copy, remove_license=True)
+
```

Comment:
You will want to call `copy_files()` before the `return`  statement.  All code after `return` is not executed since the function has already returned.

Reply:
I believe this is what is causing the tests to fail since these files are not copied over.